src/editors/containers/VideoUploadEditor/index.test.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, fireEvent, act } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import VideoUploadEditor, { VideoUploader } from '.';
+
+const mockDispatch = jest.fn();
+const mockOnUpload = jest.fn();
+
+jest.mock('react-redux', () => ({
+  useDispatch: () => mockDispatch,
+}));
+
+const defaultEditorProps = {
+  intl: {},
+};
+
+const defaultUploaderProps = {
+  onUpload: mockOnUpload,
+  errorMessage: '',
+  intl: {},
+};
+
+const renderEditorComponent = (props = defaultEditorProps) => render(
+  <IntlProvider locale="en">
+    <VideoUploadEditor {...props} />
+  </IntlProvider>,
+);
+
+const renderUploaderComponent = (props = defaultUploaderProps) => render(
+  <IntlProvider locale="en">
+    <VideoUploader {...props} />
+  </IntlProvider>,
+);
+
+describe('VideoUploadEditor', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders without errors', () => {
+    const { container } = renderEditorComponent();
+    expect(container).toMatchSnapshot();
+  });
+
+  it('renders the input field for pasting the video ID or URL', () => {
+    const { getByPlaceholderText } = renderEditorComponent();
+    const input = getByPlaceholderText('Paste your video ID or URL');
+    expect(input).toBeInTheDocument();
+  });
```

Comment:
@pkulkark I think this test is redundant.

Reply:
Ack. Will remove it.

Reply:
Done.